src/main/java/org/codehaus/mojo/license/AbstractAddThirdPartyMojo.java

```diff
@@ -113,90 +113,152 @@
-     * This is a regular expression applied to artifactIds.
+     * A regular expression (not glob pattern) used as filter to include
+     * only artifacts that have matching groupId.
+     * <p>
+     * This filter is applied on artifacts gathered as direct dependencies (and
+     * their transitive) of the projects in the reactor.
+     * <p>
+     * By default if an artifact is being included also its transitive dependencies
+     * are. To drive this behaviour refer to {@link #excludeTransitiveDependencies}.
      *
      * @since 1.1
-     */
-    @Parameter( property = "license.includedGroups" )
+     */    @Parameter( property = "license.includedGroups" )
     protected String includedGroups;
 
     /**
-     * A filter to exclude some ArtifactsIds
-     * This is a regular expression applied to artifactIds.
+     * A regular expression (not glob pattern) used as filter to exclude
+     * artifacts that have matching artifactId. Match could be also partial for
+     * example {@code ^org\.}
+     * <p>
+     * This filter is applied on artifacts gathered as direct dependencies (and
+     * their transitive) of the projects in the reactor.
+     * <p>
+     * By default if an artifact is being excluded its transitive dependencies
+     * are not automatically. To drive this behaviour refer to
+     * {@link #includeTransitiveDependencies} and
+     * {@link #excludeTransitiveDependencies}.
      *
      * @since 1.1
      */
     @Parameter( property = "license.excludedArtifacts" )
     protected String excludedArtifacts;
 
     /**
-     * A filter to include only some ArtifactsIds
-     * This is a regular expression applied to artifactIds.
+     * A regular expression (not glob pattern) used as filter to include
+     * only artifacts that have matching artifactId.
+     * <p>
+     * This filter is applied on artifacts gathered as direct dependencies (and
+     * their transitive) of the projects in the reactor.
+     * <p>
+     * By default if an artifact is being included also its transitive dependencies
+     * are. To drive this behaviour refer to {@link #excludeTransitiveDependencies}.
      *
      * @since 1.1
      */
     @Parameter( property = "license.includedArtifacts" )
     protected String includedArtifacts;
 
     /**
-     * Include transitive dependencies when checking for missing licenses and downloading license files.
-     * If this is <tt>false</tt>, then only direct dependencies are examined.
+     * Drive the artifacts gathering of the projects in the reactor build.
+     * <p>
+     * When this property is {@literal false} than only direct dependencies of
+     * projects are considered.
      *
      * @since 1.1
      */
     @Parameter( property = "license.includeTransitiveDependencies", defaultValue = "true" )
     boolean includeTransitiveDependencies;
 
     /**
-     * Exclude transitive dependencies from excluded Artifacts
+     * Drive the artifacts gathering of the projects in the reactor build.
```

Comment:
If `{@code true}` enforces excluding transitive dependencies of the excluded projects in the reactor; otherwise only direct dependencies of the reactor projects are excluded.

Reply:
> excluding transitive dependencies of the excluded projects

_of the excluded artifacts_ not projects